script/ci_setup_linux.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+apt-get update
+apt-get install -y wget git locales
+
+
+## Install CMake
+wget -nv https://github.com/Kitware/CMake/releases/download/v3.19.2/cmake-3.19.2-Linux-x86_64.sh -O setup_cmake.sh
+chmod +x setup_cmake.sh
+./setup_cmake.sh --prefix=/usr/local --skip-license
```

Comment:
This is unnecessary, GitHub provides an up to date CMake in their [virtual environment specifications](https://github.com/actions/virtual-environments).

Reply:
But how does this get into the docker container?

Reply:
`cmake` lives in `/usr/local`, so you mount it:
```yml
container:
  image: ...
  options: -v /usr/local:/host_usr_local
```
and then you make that available in the `PATH`.

Reply:
Nice, auto update for free!